src/docs/self-hosted/index.mdx

```diff
@@ -107,6 +107,31 @@ By default Sentry sends anonymous usage statistics to the Sentry team. It helps
 
 You can disable this feature by adding `SENTRY_BEACON = False` into `sentry.conf.py` file.
 
+### Errors-only Self-hosted Sentry
+
+This feature is experimental and now in beta. Starting from 24.8.0+ onwards, users will have the ability to choose between two distinct types of self-hosted Sentry deployments, tailored to their specific needs and preferences.
+
+**Errors Only**\n
+Opt for a sleek and efficient version of self-hosted Sentry that focuses on what we do best—tackling issues head-on. This lightweight option includes these features:
+1. Issues
+2. Alerts
+3. Integrations
+4. Dashboards
+5. Releases
+6. Discover
+
+**Feature Complete**\n
+This is our default version of self-hosted Sentry that is most similar feature parity wise to our SaaS product. This includes everything offered in the Errors Only version, as well as many other amazing features.
```

Comment:
This is a developer doc, not marketing. "As well as many other amazing features" is laying it one way too thick imo.

"Feature Complete Sentry includes everything available in the Errors Only version, plus the following:"

Reply:
yeah you're right, changed